telegram/_bot.py

```diff
@@ -582,8 +584,9 @@ def _insert_defaults(self, data: Dict[str, object]) -> None:
                 # Copy objects as not to edit them in-place
                 copy_list = [copy.copy(media) for media in val]
                 for media in copy_list:
-                    with media._unfrozen():
-                        media.parse_mode = DefaultValue.get_value(media.parse_mode)
+                    if hasattr(media, "parse_mode"):  # InputPaidMedia objects don't have this
+                        with media._unfrozen():
+                            media.parse_mode = DefaultValue.get_value(media.parse_mode)
```

Comment:
```suggestion
                    if not hasattr(media, "parse_mode"):
                        # InputPaidMedia objects don't have this
                        continue
                    with media._unfrozen():
                        media.parse_mode = DefaultValue.get_value(media.parse_mode)
```

actually, we could add `and not isinstance(value[0], InputPaidMedia)` in line 583. that would save us the copying of the list